tests/integration/questionnaire/test_questionnaire_endpoints.py

```diff
@@ -73,20 +73,6 @@ def test_get_thank_you_raises_404_when_questionnaire_is_not_complete(self):
         # Then we are shown a 404 page
         self.assertStatusNotFound()
 
-    def test_data_is_deleted_on_submission(self):
```

Comment:
This could check data is retained on submission.

Reply:
I don't think we need to test something hasn't been deleted and is still there